src/components/HorizontalRule/HorizontalRule.tsx

```diff
@@ -18,10 +24,19 @@ export default function HorizontalRule(
 ) {
   const { align, className, height = "2px", width = "auto" } = props;
   const styles = useStyleConfig("HorizontalRule", { align });
+  let finalHeight = height;
+
+  if (height.endsWith("%")) {
+    console.warn(
+      "For the `height` prop, use a whole number, a `px` value, a `em` value, or a `rem` value. Using the default of 2px."
```

Comment:
I don't think you need the last bit.  I also would clarify where this warning is coming from.
```
For the HorizontalRule `height` prop, a whole number, a `px` value, a `em` value, or a `rem` value mus be used.
```